pages/tentang-kami.tsx

```diff
@@ -131,10 +131,11 @@ export default function AboutPage() {
                         alt={collaborator.name}
                         className="rounded-md"
                         layout="fill"
-                        loader={({ src, width }: ImageLoaderProps) => {
+                        loader={({ src }: ImageLoaderProps) => {
                           return cloudinaryLoader({
                             src,
-                            width,
+                            // Force to show image in width 200 for smaller size
+                            width: 200,
```

Comment:
Your PR keeps undoing the changes from the previous PRs. I'm wondering how you did it.

Please make sure that there's no changes other than your own changes reflected in this PR. Thanks!